src/modules/i18n/components/LangSwitchToolbar.vue

```diff
@@ -1,58 +1,40 @@
 <template>
-    <div class="lang-switch-toolbar p-1">
-        <button
-            v-for="lang in languages"
-            :key="lang"
-            class="btn btn-sm"
-            :class="{
-                'btn-light': lang !== currentLang,
-                'btn-primary': lang === currentLang,
-            }"
-            :title="lang.toUpperCase()"
-            :data-cy="'menu-lang-' + lang"
-            @click="changeLang(lang)"
-        >
-            {{ lang.toUpperCase() }}
-        </button>
+    <div
+        :class="{
+            'lang-switch-menu': !isDesktopMode,
+            'p-1': !isDesktopMode,
+            'lang-switch-header': isDesktopMode,
+            'me-2': isDesktopMode,
```

Comment:
you can combine many classes into one statement
```
:class="{
            'lang-switch-menu p-1': !isDesktopMode,
            'lang-switch-header me-2': isDesktopMode,
```

Reply:
👍🏼